artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/converter/AmqpCoreConverter.java

```diff
@@ -220,11 +221,11 @@ public static ICoreMessage toCore(AMQPMessage message, CoreMessageObjectPools co
          }
       }
 
-      result.getInnerMessage().setReplyTo(message.getReplyTo());
       result.getInnerMessage().setDurable(message.isDurable());
       result.getInnerMessage().setPriority(message.getPriority());
       result.getInnerMessage().setAddress(message.getAddressSimpleString());
       result.getInnerMessage().setRoutingType(message.getRoutingType());
```

Comment:
isn’t this needed to mark it

Reply:
this was removing whatever I had set before causing part of the issue. This is previously set on the converter, and this was removing the property.

Reply:
and the test was asserting the case.